File: 백준/Bronze/7891. Can you add this？/Can you add this？.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        long sum = 0;
        for(int i = 0; i < n; i++){
            long x = sc.nextLong();
            long y = sc.nextLong();
            sum = x + y;
            System.out.println(sum);
        }
    }
}

File: 백준/Bronze/7891. Can you add this？/Can you add this？.c
#include<stdio.h>
int main(void){
    int T;
    scanf("%d",&T);
    long sum = 0;
    for(int i = 0; i < T; i++){
        long a,b;
        scanf("%ld%ld",&a,&b);
        sum = a + b;
        printf("%ld\n",sum);
    }
    return 0;
}